nvidia-driver-installer/ubuntu/entrypoint.sh

```diff
@@ -113,6 +117,11 @@ configure_nvidia_installation_dirs() {
 download_nvidia_installer() {
   echo "Downloading Nvidia installer..."
   pushd "${NVIDIA_INSTALL_DIR_CONTAINER}"
+  if [[ -e "${NVIDIA_INSTALLER_RUNFILE}" && -z "${NVIDIA_INSTALLER_OVERWRITE:-}" ]]; then
+    echo "Using the existing \"${NVIDIA_INSTALLER_RUNFILE}\""
+    echo 'set NVIDIA_INSTALLER_OVERWRITE=1 to replace it with a fresh copy'
```

Comment:
I think we should put this in a flag, instead of relying on the environment variable